@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -441,19 +459,123 @@ export class TextStyleCache {
                     style.wrappingMode === "Word"
                         ? WrappingMode[style.wrappingMode]
                         : WrappingMode.Word,
-                verticalAlignment:
-                    style.vAlignment === "Above" ||
-                    style.vAlignment === "Center" ||
-                    style.vAlignment === "Below"
-                        ? VerticalAlignment[style.vAlignment]
-                        : VerticalAlignment.Center,
-                horizontalAlignment:
-                    style.hAlignment === "Left" ||
-                    style.hAlignment === "Center" ||
-                    style.hAlignment === "Right"
-                        ? HorizontalAlignment[style.hAlignment]
-                        : HorizontalAlignment.Center
+                verticalAlignment,
+                horizontalAlignment,
+                placements
             }
         };
     }
 }
+
+function parseAlignmentAndPlacements(
```

Comment:
it would be good to cover function (along with its auxiliary functions below) with a unit test, either testing the function directly or indirectly through a unit test of the TextStyleCache class.